docs/TrafficSplitting.md

```diff
@@ -0,0 +1,166 @@
+# Traffic Splitting
+
+kn can help you control which revisions get traffic in your Knative Service.
+
+A Knative Service has a traffic mapping, which is a mapping of revisions of the Service to percent amounts of traffic, along with the ability to create unique URLs for particular revisions. It also has the ability to assign traffic to the latest Revision, which can change as new revisions are created.
+
+## Operations
+
+`kn` supports the following traffic operations on traffic block of a Service as part of `kn service update` command.
+
+### 1. Split Traffic:
+
+Every time you update the Service's template, a new Revision is created with 100% traffic routing to it. With every update to the Configuration of the
+Service, a new Revision is created with Service route pointing all the traffic to the latest ready Revision by default.
+However, we can change this behavior by defining which Revision gets what portion of the traffic.
+For example, you might want to carefully roll out 10% of traffic to your new Revision before putting all traffic on it.
+
+Updating the traffic is performed by `kn service update` with the `--traffic` flag.
+
+- **`--traffic RevisionName=Percent`**
+  - `--traffic` flag requires two values separated by a `=`
+  - `RevisionName` string refers to name of Revision and `Percent` integer denotes the traffic portion to assign for this Revision
+  - `Percent` integer denotes the traffic portion to assign to the this Revision
```

Comment:
Much of this information is already in the help for the sub commands

Reply:
I think we'll need to keep it at both place for completeness ?